nixos/modules/services/audio/mpdscheduler.nix

```diff
@@ -0,0 +1,59 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let 
+  cfg = config.services.mpdscheduler;
+in {
+  options.services.mpdscheduler = {
+    enable = mkEnableOption "mpdscheduler service";
+    host = mkOption {
+      type = types.str;
+      description = "the mpd host to connect to";
```

Comment:
Start with a capital and end with a period please.

Reply:
Done.